pkg/workflowengine/activities/cesr_test.go

```diff
@@ -148,6 +148,14 @@ func TestCESRValidate_Execute(t *testing.T) {
 	unix.Uname(&utsname)
 	arch := string(bytes.Trim(utsname.Machine[:], "\x00"))
 
+	// Map common uname names to release arch names
+	switch arch {
+	case "x86_64":
+		arch = "amd64"
+	case "aarch64":
+		arch = "arm64"
+	}
```

Comment:
ahaha ok! maybe you prefere to use something like this: https://github.com/dyne/lsget/blob/main/.goreleaser.yaml#L31-L44 in the et-tu-cesr release process.

BTW this should not be part of this PR ;p